### DISCUSSION TITLE
HTTP2 via reverse proxy has a concurrent request limit of 5

### DISCUSSION THREAD

**Discussion by lahsivjar** [grafana/grafana#38803] [Platform]
Currently, the maximum number of concurrent requests that are made from the frontend is controlled by the protocol configured in grafana configs ([code reference](https://github.com/grafana/grafana/blob/ba76877/pkg/api/frontendsettings.go#L248)).

Due to this, a setup involving HTTP2 via reverse proxy with grafana running on http protocol also gets the lower limit.

A possible solution would be having an option to disable limits altogether as suggested in [this comment](https://github.com/grafana/grafana/pull/26947#issuecomment-680107026).

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
